templates/post.php

```diff
@@ -1,19 +1,16 @@
 <?php
-$wwwlib = sprintf(
-    "//%s/%sresources/", 
-    htmlentities($_SERVER['SERVER_NAME']), 
-    htmlentities($this->data['baseurlpath'])
-);
+use SimpleSAML\Utils\HTTP;
 ?>
+
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <title>POST data</title>
-    <script type="text/javascript" src="<?php echo $wwwlib;?>post.js"></script>
+    <script type="text/javascript" src="<?php echo HTTP::getBaseURL();?>/resources/post.js"></script>
```

Comment:
Ah, the trailing slash was my mistake, I should have read the documentation better and I shall fix that one.

As for the `$this->data['baseurlpath']` comment; this one only seems to give the relative path to the public folder on the server, whereas `HTTP::getBaseUrl()` gives the fully qualified domain name. This last one is important when during SimpleSamlPHP install a symlink is created for the packages `simplesamlphp/www` directory. Or am I missing something here?